errcheck/errcheck.go

```diff
@@ -75,35 +74,18 @@ type UncheckedError struct {
 	FuncName string
 }
 
-// UncheckedErrors is returned from the CheckPackage function if the package contains
+// Result is returned from the CheckPackage function if the package contains
 // any unchecked errors.
 // Errors should be appended using the Append method, which is safe to use concurrently.
-type UncheckedErrors struct {
+type Result struct {
 	mu sync.Mutex
```

Comment:
Is this still needed now that the checker has no concurrency?

Reply:
Probably not. I'm in favour of removing it and leaving the concurrency to the caller.